src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -2,12 +2,14 @@ package ar.edu.unahur.obj2.semillasAlViento
 
 class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
   val plantas = mutableListOf<Planta>()
+  //Redundancia Minima ya que es innecesario una variable cantidad de plantas
   var cantidadPlantas = 0
```

Comment:
La cantidad de plantas se puede sacar directamente de la lista de plantas
```suggestion
  fun cantidadPlantas() = plantas.size
```